### ISSUE TITLE
Blockquotes push content outside of boundaries

### ISSUE THREAD

**Issue by firasdib** [silverbulletmd/silverbullet#432]
The line where a blockquote is started, using `>`, has a negative text indent which pushes the text outside of the box.

I've seen similar behavior with the titles (`# title`), which leads me to believe this was a design choice, but perhaps a bit of an oversight in this case. 

![image](https://github.com/silverbulletmd/silverbullet/assets/1335165/e3b50fc2-4b51-4ee8-9d41-b7880fc38b51)

**Comment by zefhemel** [comment]
Yeah, blockquotes are a bit of a mess. Have to think how to style them reliably supporting all markup that can happen inside them...
